app/views/autograders/_form.html.erb

```diff
@@ -1,4 +1,28 @@
-<%= form_for [@course, @assessment, @autograder], builder: FormBuilderWithDateTimeInput do |f| %>
+<% content_for :javascripts do %>
+	<script>
+		$(function() {
+		  $(document).on('change', ':file', function() {
+		    var input = $(this),
+		    	numFiles = input.get(0).files ? input.get(0).files.length : 1,
+		      label = input.val().replace(/\\/g, '/').replace(/.*\//, '');
+		    	input.trigger('fileselect', [numFiles, label]);
+		  });
+
+		  $(document).ready( function() {
+		    $('#autograder_makefile:file').on('fileselect', function(event, numFiles, label) {
```

Comment:
Why not just use the default [Materialize File Input](http://materializecss.com/forms.html)? It does this for you?